matrix-sdk/src/main/java/org/matrix/androidsdk/db/MXMediasCache.java

```diff
@@ -599,7 +599,7 @@ public String saveBitmap(Bitmap bitmap, String defaultFileName) {
             File file = new File(getFolderFile(null), filename);
             FileOutputStream fos = new FileOutputStream(file.getPath());
 
-            bitmap.compress(Bitmap.CompressFormat.JPEG, 100, fos);
+            bitmap.compress(Bitmap.CompressFormat.PNG, 100, fos);
```

Comment:
I think there is an issue because we use mime type "image/jpeg" in `RoomMediaMessagesSender`.
Also filename is `.jpg` in this method.
Maybe we can considere keeping JPG for this PR and try to do better on another PR: keep PNG or JPG depending on the original image format.